### DISCUSSION TITLE
Individual foreground colors for each 16 colors

### DISCUSSION THREAD

**Discussion by jude7733** [eylles/pywal16#64] [Ideas]
_HI, I'm new here_

**motivation** - unreadable text when styling waybar with different 16 colors

**IDK** - if this feature already exist or if it's part of project scope

**Need** - a minimum contrast ratio of 4.5:1 for normal text

**Feature** - somehow generate readable font colors (eg @foreground1,... @foreground15) associated with each colors

**Comment by eylles** [comment]
with some of the recent additions and plans for 3.9.0 there's a good chance of considering this idea and a plan to implement it.
